wled00/FX.cpp

```diff
@@ -5100,17 +5104,17 @@ uint16_t mode_2DFrizzles(void) {                 // By: Stepko https://editor.so
   const int cols = SEGMENT.virtualWidth();
   const int rows = SEGMENT.virtualHeight();
 
-  SEGMENT.fadeToBlackBy(16);
+  SEGMENT.fadeToBlackBy(18);
```

Comment:
This will cause *different looking* effectand may case playlists to no longer function as intended. Please consider backwards compatibility.